### DISCUSSION TITLE
Mixergy Servers Down?

### DISCUSSION THREAD

**Discussion by barryperfect** [tomasmcguinness/homeassistant-mixergy#67] [Q&A] [answered]
Not specifically a question about the integration, but has anyone else had a problems with accessing the Mixergy servers since 10.30 am this morning (29th Dec)? 

Connectivity on the tank itself seems fine and I can control the tank via the remote, but I just get a spinning wheel in the Mixergy App and a charge of -1% and -1 deg C Temp in the integration.

Mixergy support are off until 4th January

**Comment by DirkFace** [comment]
Just checked my app, same for me.

**Comment by tomasmcguinness** [comment]
I checked earlier and my Home Assistant was showing data. I've just looked at the app and it's not connecting.

I'm seriously thinking of how to reverse engineer that little wired control unit to make the tank controllable locally :)

**Comment by barryperfect** [comment, marked as answer]
Looks like it is back working again, in the app and the integration, suspect someone had to turn it off and back on again at Mixergy HQ :-)
